apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtaksvurderingRepository.kt

```diff
@@ -317,6 +234,35 @@ class VedtaksvurderingRepository(
         }
     }
 
+    fun hentVedtakMedUtbetalingForInntektsaar(
+        inntektsaar: Int,
+        sakType: SakType? = null,
+        tx: TransactionalSession? = null,
+    ): List<Vedtak> {
+        val hentVedtak =
+            """
+            SELECT * FROM vedtak v
+            JOIN utbetalingsperiode u ON v.id = u.vedtakid
+            WHERE EXTRACT(YEAR FROM u.datofom) = :aar
```

Comment:
Jeg er ikke helt sikker, men tror vi kan få saker som ikke har utbetaling i inntektsåret her -- jeg vet f.eks. ikke om vi sletter utbetalingsperiodene til vedtak hvis vi går fra innvilgelse til avslag -- og så vil opphør bakover i tid kunne potensielt "nulle ut" radene. 

Ikke at det er noe veldig vanlig forekommende, men det er verdt å tenke på med denne måten å hente alle sakene på

Reply:
Tenker kanskje denne står seg, og at vi i oppretting av Etteroppgjør for en sak gjør de nødvendige sjekkene.